docs/06-concepts/18-testing/02-the-basics.md

```diff
@@ -228,17 +238,17 @@ In production, the transaction call will throw if any database exception happene
 
 The run mode that Serverpod should be running in. Defaults to `test`.
 
-### `enableSessionLogging`
+### `serverpodLoggingMode`
 
-Wether session logging should be enabled. Defaults to `false`.
+The logging mode used when creating Serverpod. Defaults to `ServerpodLoggingMode.normal`.
 
-### `applyMigrations`
+### `serverpodStartTimeout`
 
-Wether pending migrations should be applied when starting Serverpod. Defaults to `true`.
+The timeout to use when starting Serverpod, which connects to the database among other things. Defaults to `Duration(seconds: 30)`.
 
 ### `testGroupTagsOverride` {#test-group-tags-override-configuration}
```

Comment:
Question: Should `Override` be removed from this flag now that it is not an override but the actual test group used?

Reply:
Hmm, doesn't override imply that it replaces the existing one? 🤔

Reply:
Only specifying `testGroupTags` would seem more ambiguous to me I think.

Reply:
You are correct! My logic is not correct regarding this. The override keyword should be used becuase that is what we are doing :)